bench/wtperf/runners/wtperf_run.sh

```diff
@@ -39,18 +39,18 @@ rm -f $outfile
 echo "Parsed $# args: test: $wttest runmax: $runmax args: $wtarg" >> $outfile
 
 # Each of these has an entry for each op in ops below.
-avg=(0 0 0 0)
-max=(0 0 0 0)
-min=(0 0 0 0)
-sum=(0 0 0 0)
+avg=(0 0 0 0 0)
+max=(0 0 0 0 0)
+min=(0 0 0 0 0)
+sum=(0 0 0 0 0)
 # Load needs floating point and bc, handle separately.
-loadindex=5
+loadindex=6
 avg[$loadindex]=0
 max[$loadindex]=0
 min[$loadindex]=0
 sum[$loadindex]=0
-ops=(read insert update truncate)
-outp=("Read count:" "Insert count:" "Update count:" "Truncate count:")
+ops=(read insert update modify truncate)
+outp=("Read count:" "Insert count:" "Update count:" "Modify count:" "Truncate count:")
```

Comment:
Alphabetize (including fixing the earlier miss of that).

Reply:
corrected.